protoc-gen-grpc-gateway/gengateway/template.go

```diff
@@ -266,7 +306,11 @@ var (
 		grpclog.Infof("Failed to start streaming: %v", err)
 		return nil, metadata, err
 	}
-	dec := marshaler.NewDecoder(req.Body)
+	body, berr := ioutil.ReadAll(req.Body)
```

Comment:
I had to add this to the `bidi-streaming-request-func` because I added the `ioutil` and `bytes` imports in the generator code above. Certain cases would cause compile errors for unused imports.